modules/phase_field/include/ics/PolycrystalReducedIC.h

```diff
@@ -46,6 +46,10 @@ class PolycrystalReducedIC : public InitialCondition
   virtual void initialSetup();
 
 protected:
+  bool assignColors(const std::vector<std::vector<bool> > & adjacency_matrix, std::vector<unsigned int> & colors, unsigned int grain) const;
```

Comment:
Ok, I guess it ended up being exposed to the end user. I was kind of hoping to avoid that, but it's a necessary evil for the flexibility I want.